src/components/Reviews.jsx

```diff
@@ -5,40 +5,71 @@ import { Link, useParams, useSearchParams } from 'react-router-dom';
 const Reviews = () => {
 const [isLoading, setIsLoading] = useState(true);
 const [reviews, setReviews] = useState([]);
-const [category, setCategory] = useSearchParams();
-const categoryType = category.get('category')
+const [searchParams, setSearchParams] = useSearchParams();
+const sortbyQuery = searchParams.get('sort_by');
+const orderQuery = searchParams.get('order');
+
 
 useEffect(() => {
     setIsLoading(true);
-    getReviews(categoryType).then(({ reviews }) => {
+    getReviews(sortbyQuery, orderQuery).then(({ reviews }) => {
         setReviews(reviews);
         setIsLoading(false);
     });
-}, [categoryType]);
+}, [sortbyQuery, orderQuery]);
 
 if (isLoading) {
     return <p className="Loading">Loading...</p>;
 };
 
+const setSearch = (sort_by, order) => {
+    const newParams = new URLSearchParams(searchParams);
+    if (sort_by) {newParams.set('sort_by', sort_by)};
+    if(order) {newParams.set('order', order)};
```

Comment:
When writing one line code blocks you don't need the curly brackets. 
e.g. 
```js
if (sort_by) newParams.set('sort_by', sort_by);
```
If you add the curly brackets then you should put the code on a new line.